File: Data Structures & Algorithms/max-consecutive-ones/submission-2.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count = 0;
        int most = 0;
        for (int num: nums) {
            if (num == 0) {
                most = std::max(count, most);
                count = 0;
            }
            else {
                count += 1;
            }
        }
        return std::max(count, most);
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums.sort()
        if not nums:
            return 0
        winner = 0
        current = 1
        for first, second in zip(nums[:-1], nums[1:]):
            if first == second:
                continue
            if second == first + 1:
                current += 1
                # winner = max(winner, current)
            else:
                winner = max(winner, current)
                current = 1
        return max(winner, current)
            